packages/front-end/components/Experiment/EditMetricsForm.tsx

```diff
@@ -40,75 +47,95 @@ const EditMetricsForm: FC<{
   const { apiCall } = useAuth();
 
   return (
-    <Modal
-      autoFocusSelector=""
-      header="Edit Metrics"
-      size="lg"
-      open={true}
-      close={cancel}
-      submit={form.handleSubmit(async (value) => {
-        await apiCall(`/experiment/${experiment.id}`, {
-          method: "POST",
-          body: JSON.stringify(value),
-        });
-        mutate();
-      })}
-      cta="Save"
-    >
-      <div className="form-group">
-        <label className="font-weight-bold mb-1">Goal Metrics</label>
-        <div className="mb-1 font-italic">
-          Metrics you are trying to improve with this experiment.
-        </div>
-        <MetricsSelector
-          selected={form.watch("metrics")}
-          onChange={(metrics) => form.setValue("metrics", metrics)}
-          datasource={experiment.datasource}
-          autoFocus={true}
+    <>
+      {upgradeModal && (
+        <UpgradeModal
+          close={() => setUpgradeModal(false)}
+          reason="Override metric conversion window"
```

Comment:
I would make `window` plural and add a period at the end.

```suggestion
          reason="Override metric conversion windows."
```